src/mp4box/meta.rs

```diff
@@ -2,13 +2,38 @@ use std::io::{Read, Seek};
 
 use serde::Serialize;
 
+use crate::mp4box::hdlr::HdlrBox;
 use crate::mp4box::ilst::IlstBox;
 use crate::mp4box::*;
 
-#[derive(Debug, Clone, PartialEq, Eq, Default, Serialize)]
-pub struct MetaBox {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub ilst: Option<IlstBox>,
+#[derive(Debug, Clone, PartialEq, Eq, Serialize)]
+#[serde(tag = "hdlr")]
+#[serde(rename_all = "lowercase")]
+pub enum MetaBox {
```

Comment:
Should we also implement the Mp4Box trait for meta (and child boxes)? So I can access `mp4.moov.meta` for example.

Reply:
good idea. i already had written another commit 669edcf to be able to write meta and udta boxes but was waiting for this PR to merge before opening another. I'll just push it to this branch.